components/landing/CTASection.tsx

```
@@ -0,0 +1,54 @@
+import Link from 'next/link'
+import { Button } from '@/components/ui/button'
+import { GradientBackground } from './GradientBackground'
+import { useIntersectionObserver } from '@/hooks/useIntersectionObserver'
+
+export function CTASection() {
+  const { ref, isVisible } = useIntersectionObserver({ threshold: 0.3 })
+
+  return (
+    <GradientBackground variant="cta" className="py-12 md:py-16 lg:py-20">
+      <section 
+        ref={ref as any} 
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)